baseplate/__init__.py

```diff
@@ -93,8 +96,90 @@ def make_tracing_client(raw_config, log_if_unconfigured=True):
     )
 
 
+def crash_reporter_from_config(raw_config, module_name):
```

Comment:
Right now it's being used to figure out where the git repo is for doing `raven.fetch_git_sha` on for the `release` tag in Sentry. I used `__name__` instead of something else since I thought it might be more future-proofed for something like (from OP):

>  since potentially this function could do fancy stuff down the line with the module name and the include_paths directive